website_sale_charge_payment_fee/models/payment_acquirer.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 AITIC S.A.S
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+import odoo.addons.decimal_precision as dp
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = "payment.acquirer"
+
+    charge_fee = fields.Boolean(
+        "Fee charged to customer",
+        help="An extra fee line will be added to online order when using this "
+        "payment method",
+    )
+    charge_fee_description = fields.Text("Fee Description")
+    charge_fee_product_id = fields.Many2one("product.product", string="Fee Product")
+    charge_fee_tax_ids = fields.Many2many("account.tax", string="Taxes")
```

Comment:
```suggestion
```
I think we should delete the taxes here, although I am not sure about the intention by the original author. IMO the taxes should be proposed from the product, just like any other products.

Reply:
@eLBati Can you perhaps comment on this please?

Reply:
@yostashiro I don't remember if this wanted to cover any specific requirement, but I don't think so.
I agree with your approach.
Thanks

Reply:
@eLBati Thanks for your comment!